requirements/main.txt

```diff
@@ -15,7 +15,9 @@
 bs4
 click < 8.2.0  # https://github.com/ai-dynamo/dynamo/issues/1039
 datasets<4  # lcb problem with datasets 4.0.0
+ddgs # Needed for BFCLv4
```

Comment:
this requirement is causing build issues--the dockerfile cannot be built with it.